ui/src/app/components/runtime-configuration-panel/disk-selector.tsx

```diff
@@ -53,142 +49,102 @@ export const DiskSelector = ({
           </StyledExternalLink>
         </FlexRow>
       </FlexColumn>
-      {computeType === ComputeType.Standard && (
-        <WarningMessage>
-          <AoU /> will now only support reattachable persistent disks as the
-          storage disk option for Standard VMs and will discontinue standard
-          disks. You will continue to use standard disks with Dataproc clusters.
-          Refer to the
-          <a
-            href={
-              'https://support.researchallofus.org/hc/en-us/articles/5140493753620-Persistent-Disk'
-            }
-            target='_blank'
-          >
-            {' '}
-            article{' '}
-          </a>{' '}
-          to learn more.
-        </WarningMessage>
-      )}
-      <TooltipTrigger
```

Comment:
removed this from the standard-VM block, because we will no longer show the option for a standard-disk